packages/components/package.json

```diff
@@ -13,7 +13,8 @@
     "@styled-system/color": "^5.1.2",
     "@styled-system/css": "^5.0.23",
```

Comment:
Looks like this hasn't been updated in the `css-package` branch yet...

Reply:
Rebased!